helloworld/helloworldTests/helloworldTests.swift

```diff
@@ -25,7 +25,6 @@ class HelloworldTests: XCTestCase {
         // This is an example of a functional test case.
         // Use XCTAssert and related functions to verify your tests produce the correct results.
         
-        XCTFail("TESTING FAILURE!")
     }
     
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/marianoabdala/HelloWorld/blob/3817478880ed8b8bf3d7781b44405c2d4d6277ed/helloworld/helloworldTests/helloworldTests.swift#L29"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>